src/main/java/org/mskcc/cbio/oncokb/service/SlackService.java

```diff
@@ -329,30 +329,6 @@ private LayoutBlock buildUserIdBlock(UserDTO userDTO) {
         return ContextBlock.builder().elements(elements).blockId(USER_ID.getId()).build();
     }
 
-    private List<LayoutBlock> buildWarningBlocks(UserDTO userDTO) {
```

Comment:
I think we still need the warning blocks here, just not he warning about the MSK user.

Reply:
The content needs to be updated to search the company info.